modules/meb_api/lib/dgi/automation/claimant_response.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'dgi/response'
+
+module MebApi
+  module DGI
+    module Automation
+      class ClaimantResponse < MebApi::DGI::Response
+        attribute :claimant_id, Integer
+        attribute :suffix, String
+        attribute :date_of_birth, String
+        attribute :first_name, String
+        attribute :last_name, String
+        attribute :middle_name, String
+        attribute :notification_method, String
+        attribute :preferred_contact, String
+        attribute :address_line_1, String
+        attribute :address_line_2, String
+        attribute :city, String
+        attribute :zipcode, String
+        attribute :address_type, String
+        attribute :email_address, String
+        attribute :mobile_phone_number, String
+        attribute :home_phone_number, String
+        attribute :country_code, String
+        attribute :state_code, String
+
+        def initialize(status, response = nil)
```

Comment:
This method is causing Rubocop errors regarding complexity and length. Some thoughts we've had are that because the data is already being massaged in the third party service that we are calling and the response has so many attributes that it would be ideal if we didn't have to fetch each value and we could just dump the entire json response. Is there an existing pattern we can follow to adopt this approach or what is recommended as far as best practice here?

Reply:
I think `dig` would help with complexity here. And for method length, would it work to extract that hash to an ` attributes ` method?